### PR TITLE
Nashville Chicken Sandwich

### PR DESCRIPTION
**Pull request by marttoygoy4**

### CODE REVIEW

**Comment by endlesslake** [comment]
The file name can't have spaces or caps. It should be `nashville-chicken.md`.
There is an unformatted duplicate link in your contribution section.
You need to add tags.

File: src/Nashville Chicken.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+4. Once your oil has reached the temp of 350, go ahead and fry 3-4 pieces of chicken at one time. You are going to fry for 7 mins (flipping at the midway point) and then pull the chicken and let sit for 5 mins then refry for the second time 3 mins, this depends on the thickness of the chicken.
+5. Now that your chicken is fried make sure you have a wire rack with a sheet pan under it so the crispy chicken can sit and rest on. During this resting time, you are going to add your Nashville Hot Chicken seasoning together with 1 1/2 cups of the cooking oil you just used. It will simmer but quickly whisk until it comes still. Now you want to brush that amazing sauce all over the crispy chicken or dunk it. 
+6. Have some toasted buns ready and create your sandwich. First, you want to sauce your top and bottom buns, add shredded romaine lettuce, chicken patty, tomato and then complete with the top bun for an amazingly delicious creation.
+7. Feed the family and enjoy! This one gets spicy, >E  N  J  O Y<
+
+## Contribution
+
+
+https://grillnationbbq.com/wprm_print/1538
+
```
Comment (hallzy) [inline src/Nashville Chicken.md:57]:
```suggestion
```

Comment (hallzy) [reply src/Nashville Chicken.md:57]: This link exists below

File: src/Nashville Chicken.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- BBQ-Nation _Credit_ [website](https://grillnationbbq.com/wprm_print/1538)
```
Comment (hallzy) [inline src/Nashville Chicken.md:58]: Add some tags below

**Comment by hallzy** [comment]
The file name should also be lowercase: `nashville-chicken.md`

**Comment by marttoygoy4** [comment]
> The file name should also be lowercase: `nashville-chicken.md`

Kek I am a literal retard

**Pull request merged**